TypeInjections/Tool/Tool.fs

```diff
@@ -66,12 +66,8 @@ module Tool =
         let commitRepo = newRepo.Lookup<Commit>(hash)
         // checkout repo at given commit
         Commands.Checkout(newRepo, commitRepo) |> ignore
-        // ensure checked out repo has dafny file(s)
-        checkInputs [ outputPath ]
-
-
 
-    let runGit (git: GitOptions) : unit =
+    let runGit (git: GitOptions) : string * string * string * string option =
```

Comment:
What's this return value? Can we introduce a type for it?

Reply:
This return value is the parsed `option` values that the user inputted. For `runGit` and `runlocal` it is:
string = location of "old"
string = location of "new"
string = location of output folder
string option = Optional .typecartignore file path

Since these values are used the same way, I can introduce a `parsedOption` type to make it more readable